README.md

```diff
@@ -117,8 +92,6 @@ fun SampleView() {
 
 ### Advanced usage
 
-Retained includes support for Kotlin coroutines, `SavedStateHandle`, and more.
```

Comment:
Removed this in favor of going more in details on this below. I personally really like this part of this library and thought those things deserved to be shown more here.